### PR TITLE
Fetch

### PR DESCRIPTION
**Pull request by Jibanandamht**

### CODE REVIEW

File: fetching.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+                                $fetch_sql = "SELECT * FROM `emails`";
```
Comment (Jibanandamht) [inline fetching.php:2]: @probhatmht

Comment (Jibanandamht) [reply fetching.php:2]: done

**Review by Jibanandamht — COMMENTED** [review]
@probhatmht

File: fetching.php (hunk 1 above)
Comment (Probhatmht) [reply fetching.php:2]: ok

**Review by Probhatmht — CHANGES_REQUESTED** [review]
change ?

**Review by Probhatmht — APPROVED** [review]
done

**Review by Probhatmht — APPROVED** [review]
done

**Comment by Electro1212** [comment]
Shhaaar esob kiiiiiii?

**Pull request merged**